mail-toaster.sh

```diff
@@ -535,6 +539,7 @@ has_data_fs()
 		postgres ) return 0;;
 		haproxy )  return 0;;
 		minecraft ) return 0;;
+		php7 )     return 0;;
```

Comment:
Since PHP7 is new a lot of old code isn't supported yet. Most modern are (Wordpress and roundcube).
Running PHP7 has a lot of benefits for those sites, big speed improvement (just do a quick google).
Some customers have own build websites and you can't force them to upgrade to PHP7, so on another server I have a separate jail which runs indeed PHP5.

I'm not a fan of a jail for every customer (for a website), since then the overhead and maintenance would be too much imo. (run 15 wordpress sites for example)

So if people want to run PHP5 it's just a matter of changing ip in nginx and point to php5-fpm